Swiftification/Range.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2015 Hilton Campbell
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+import Foundation
+
+public extension Range {
+
+    /// Returns the Range as an Array
+    func toArray() -> [Element] {
+        return self.map { $0 }
+    }
+    
+    /// Returns a random item
+    func random() -> Element {
+        return toArray().random()
```

Comment:
Yeah, but then `0...10000000.random()` creates a huge array. `Range` is a `ForwardIndexType` with `distanceTo:`... could probably use that (http://swiftdoc.org/v2.1/protocol/ForwardIndexType/).